app/Http/Controllers/AddressController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\{Address,User,Area};
+use Exception;
+
+class AddressController extends Controller
+{
+    public function userAddresses($id){
+        return response()->json(User::find($id)->addresses, 200);
```

Comment:
1. Use the "address" model directly instead of using the "user", current approach doesn't handle the possibility of having an incorrect user id.
2. For all your responses across the controller, make sure that you are returning your data inside a variable to make it easier for whoever is going to be calling the endpoints from the client side.